tests/AutoReview/FixerFactoryTest.php

```diff
@@ -279,16 +280,78 @@ function (array $case) use ($self) {
         );
     }
 
+    public function testPriorityIntegrationTestFilesAreListedPriorityCases()
+    {
+        $priorityCases = $this->provideFixersPriorityCases();
+
+        foreach (new \DirectoryIterator($this->getIntegrationPriorityDirectory()) as $candidate) {
+            if ($candidate->isDot()) {
+                continue;
+            }
+
+            $fileName = $candidate->getFilename();
+            $this->assertTrue($candidate->isFile(), sprintf('Expected only files in the priority integration test directory, got "%s".', $fileName));
+            $this->assertFalse($candidate->isLink(), sprintf('No (sym)links expected the priority integration test directory, got "%s".', $fileName));
+            $this->assertSame(
+                1,
+                preg_match('#^([^,\s]+),([^,\s\d]+)(?:_\d{1,3})?\.test$#', $fileName, $matches),
+                sprintf('File with unexpected name "%s" in the priority integration test directory.', $fileName)
+            );
+
+            $fixerName1 = $matches[1];
+            $fixerName2 = $matches[2];
+
+            $caseFound = false;
+            foreach ($priorityCases as $priorityCase) {
```

Comment:
I would process the `$priorityCases` array before the main `foreach` to avoid this nested `foreach`.